Gemfile.lock

```diff
@@ -31,14 +31,14 @@ GEM
       guard (~> 2.1)
       guard-compat (~> 1.1)
       rspec (>= 2.99.0, < 4.0)
-    listen (3.2.1)
+    listen (3.8.0)
       rb-fsevent (~> 0.10, >= 0.10.3)
       rb-inotify (~> 0.9, >= 0.9.10)
     log_buddy (0.7.0)
     lumberjack (1.2.8)
     method_source (1.0.0)
-    mongo (2.13.0)
-      bson (>= 4.8.2, < 5.0.0)
+    mongo (2.19.3)
```

Comment:
I got the following error while running `bundle exec rake`.

> /Users/masato/.rbenv/versions/3.2.2/lib/ruby/gems/3.2.0/gems/rspec-core-3.9.2/exe/rspec: warning: Exception in finalizer #<Proc:0x000000010488e6b8 /
+Users/masato/.rbenv/versions/3.2.2/lib/ruby/gems/3.2.0/gems/mongo-2.13.0/lib/mongo/server.rb:285>
> /Users/masato/.rbenv/versions/3.2.2/lib/ruby/3.2.0/timeout.rb:101:in `new': can't alloc thread (ThreadError)

So I updated mongo gem with the following command.

```
$ bundle update mongo --conservative
```